### PR TITLE
Interactive list

### PR DESCRIPTION
**Pull request by Gracetemitope**
In this branch, I implemented the following:

Drag and drop functionality
Update completed status (true/false)
Add todos functionality

### CODE REVIEW

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [ ] When I drag and drop an element, this does it, but if I refresh the page the local storage doesn't save the new position. I kindly invite you to fix this.
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/dragAble.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    const draggableElements = [...document.querySelectorAll('.draggable:not(.drag-sort-active)')];
+    const draggable = document.querySelector('.drag-sort-active');
+    const container = document.getElementById('todo-list');
+    const afterElement = draggableElements.reduce((closest, child) => {
+      const box = child.getBoundingClientRect();
+      const offset = event.clientY - box.top - box.height / 2;
+      if (offset < 0 && offset > closest.offset) {
+        return { offset, element: child };
+      }
+      return closest;
```
Comment (DeliceLydia) [inline src/dragAble.js:52]: - [ ] Currently when I drag a list item still local storage doesn't update a new position when i load the page
- [ ] Need to work on the file that contain methods related to the status updates (completed: true / false). currently when I check a box it doesn't get updated from `true` to `false` or vice versa

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
# Hello @Gracetemitope , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._


### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by voscarmv — APPROVED** [review]
Hi @Gracetemitope , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**